packages/application-config/schema.json

```diff
@@ -17,10 +17,6 @@
     }
   },
   "properties": {
-    "id": {
```

Comment:
This has moved to the `env.production.applicationId` field (it wasn't used before anyway).